test/jdk/java/net/DatagramSocket/SendDatagramToBadAddress.java

```diff
@@ -24,11 +24,11 @@
 /*
  * @test
  *
- * @bug 4204320
+ * @bug 4204320 8278961
```

Comment:
Ah - can you remove 8278961 from the bug list before pushing?

Reply:
`@bug` is supposed to be used to list the bugs that the test is verifying...

Reply:
Hello Daniel,
I removed that bug id and also updated the copyright year to the new year and updated the PR. No other changes.